packages/health-check/src/checks/critical.js

```diff
@@ -122,8 +122,9 @@ async function registryWriteAndReadCheck(done) {
 
 // directServerApiAccessCheck returns the basic server api check on direct server address
 async function directServerApiAccessCheck(done) {
-  if (!process.env.SERVER_DOMAIN) {
-    return done({ up: false, errors: [{ message: "SERVER_DOMAIN env variable not configured" }] });
+  // skip if SERVER_DOMAIN is not set or it equals PORTAL_DOMAIN (single server portals)
+  if (!process.env.SERVER_DOMAIN || process.env.SERVER_DOMAIN === process.env.PORTAL_DOMAIN) {
```

Comment:
Is this correct? Shouldn't we return an error if both `SERVER_DOMAIN` and `PORTAL_DOMAIN` are empty?

Reply:
we already have a different test that makes sure PORTAL_DOMAIN points to working website

https://github.com/SkynetLabs/skynet-webportal/blob/4af101cb4ce531645116df34e28de4817a5d9c30/packages/health-check/src/checks/critical.js#L67-L69